src/server/services/about/views/service.njk

```diff
@@ -197,8 +209,17 @@
               <ul class="govuk-list govuk-!-margin-0">
                 {% for url in firstVanityUrl.urls %}
                   <li>
+                    {% if url.shuttered %}
+                       {% call appToolTip({ text: "URL is shuttered.", classes: "app-tool-tip--small" }) %}
+                       {{ appShutterIcon({ classes: "app-icon--tiny app-icon--fill-red", description: "URL is shuttered" }) }}
+                       {% endcall %}
+                     {% elif not url.enabled %}
+                       {% call appToolTip({ text: "URL has not been enabled yet. Contact #cdp-support to enable it.", classes: "app-tool-tip--small" }) %}
+                       {{ appBlockIcon({ classes: "app-icon--tiny app-icon--fill-gray", description: "URL is not enabled" }) }}
```

Comment:
Looking at this in the flesh I think this should probably be red

Reply:
![image](https://github.com/user-attachments/assets/4b3b2448-9e08-4188-b5bc-062cb42273ff)